fdbserver/worker.actor.cpp

```diff
@@ -239,19 +239,28 @@ std::vector< DiskStore > getDiskStores( std::string folder, std::string suffix,
 		DiskStore store;
 		store.storeType = type;
 
-		StringRef prefix;
-		if( StringRef( files[idx] ).startsWith( fileStoragePrefix ) ) {
+		StringRef filename = StringRef( files[idx] );
+		Standalone<StringRef> prefix;
+		if( filename.startsWith( fileStoragePrefix ) ) {
 			store.storedComponent = DiskStore::Storage;
 			prefix = fileStoragePrefix;
 		}
-		else if( StringRef( files[idx] ).startsWith( fileLogRefSpillDataPrefix ) ) {
+		else if( filename.startsWith( fileVersionedLogDataPrefix ) ) {
```

Comment:
Instead of searching for "log-V" to catch "log-V6_0", I could just always check the thing after "log-" to see if it's a version, and if so, use it ; otherwise assume it's an unversioned old-style file.

In the case of an unexpected file, I'm unsure if that'd be better behavior or not, but our UUIDs will never start with a V anyway.

Reply:
Now moot